Sources/MapboxMaps/Ornaments/OrnamentOptions.swift

```diff
@@ -50,6 +50,9 @@ public struct ScaleBarViewOptions: OrnamentOptionsProtocol, Equatable {
     public var margins: CGPoint = defaultOrnamentsMargin
     /// The default value for this property is `.adaptive`.
     public var visibility: OrnamentVisibility = .adaptive
+    /// Specifies the whether the scale bar uses the metric system.
+    /// True if the scale bar is using metric units, false if the scale bar is using imperial units.
+    public var useMetricUnits: Bool = Locale.current.usesMetricSystem
```

Comment:
Discussion: While the Bool flag works fine here and even Apple uses the same Bool in `Locale`, I would like to suggest using `enum` with concrete cases. It would increase readability as well as make some room for extensions and, potential and theoretical, support of our measurement systems (if there are any exist).

Reply:
For example, we have a pattern of access to a table of constants like `useMetricUnits ? Constants.metricTable : Constants.imperialTable`. 
We could make a function/variable in custom enum to make access even easier.